pkg/docker/cockpit-atomic-storage

```diff
@@ -420,16 +420,12 @@ def cmd_monitor():
     mon = subprocess.Popen([ "stdbuf", "-o", "L", "udevadm", "monitor", "-u", "-s", "block"],
                            bufsize=1, stdout=subprocess.PIPE)
     while mon.poll() == None:
-        (r, _, _) = select.select([ mon.stdout ], [] , [], 5)
-        got_event = False
-        if len(r) > 0:
-            if mon.stdout.readline().decode().startswith("UDEV  "):
-                got_event = True
-        # XXX - skip udev/lvm stuff on pure timeouts
-        info = get_info()
-        if info != old_info:
-            print_info(info)
-            old_info = info
+        if select.select([mon.stdout], [] , [], 5)[0] and mon.stdout.readline().decode().startswith("UDEV  "):
+            # XXX - skip udev/lvm stuff on pure timeouts
+            info = get_info()
+            if info != old_info:
+                print_info(info)
+                old_info = info
```

Comment:
@mvollmer: The original code is from your commit 3c12e322, so I'd appreciate a review of this bit to ensure this is really what you intended. TIA!